lte/gateway/c/connection_tracker/src/main.cpp

```diff
@@ -65,14 +66,17 @@ static uint32_t get_log_verbosity(const YAML::Node& config,
 }
 
 int main(void) {
-  magma::init_logging(CONNECTION_SERVICE);
 
   auto mconfig = load_mconfig();
   auto config =
       magma::ServiceConfigLoader{}.load_service_config(CONNECTION_SERVICE);
+  magma::init_logging(CONNECTION_SERVICE);
   magma::set_verbosity(get_log_verbosity(config, mconfig));
   MLOG(MINFO) << "Starting Connection Tracker";
 
+  sentry_config_t sentry_config = construct_sentry_config_from_mconfig();
+  initialize_sentry(SENTRY_TAG_CONNECTIOND, &sentry_config);
```

Comment:
we probably want to start sentry as early as possible. what are we blocked on above?